R/get_employee_file.R

```diff
@@ -10,8 +10,6 @@
 #' @return response object
 #'
 #' @examples
-#' keys <- globalEntry::get_keys()
```

Comment:
can maybe clear the whole examples section rather than just leaving with nothing after it, should clean up some notes from devtools::check() by doing this